js/components/portal-dashboard/question-area.tsx

```diff
@@ -77,4 +80,12 @@ export class QuestionArea extends React.PureComponent<IProps>{
       </div>
     );
   }
+
+  private renderImageQuestionPrompt = (drawingPrompt: string) => {
+    return (
+      <div className={css.imageQuestionPrompt}>
+        {renderHTML(drawingPrompt)}
+      </div>
```

Comment:
One minor issue is that if there is only a drawing prompt and no main prompt, then they will still see the divider line.  Not sure if that matters.

Reply:
If you had a `showDivider` boolean, then you could do:
```
<div className={`${css.imageQuestionPrompt} ${showDivider ? css.showDivider : ""}`}>
```
And add the appropriate CSS to show/hide the border